test/suites/incus_user.sh

```diff
@@ -0,0 +1,19 @@
+test_incus_user() {
+  ensure_import_testimage
+  ensure_has_localhost_remote "${INCUS_ADDR}"
+
+  incus-user --group nogroup &
+  USER_PID="$!"
+  while :; do
+    [ -S "${INCUS_DIR}/unix.socket.user" ] && break
```

Comment:
(sorry) well, that will spin pretty hard.  Can you add a sleep 0.5s in there or something?  The idea isn't to get past this as fast as possible, but rather to avoid getting to the later commands before it's ready.

Reply:
oops, yeah, I meant for a 0.5 :)

Reply:
Fixed